.github/workflows/ci.yml

```diff
@@ -252,43 +165,24 @@ jobs:
           name: Build output
           path: artifacts/build
 
-      - name: Integration Test - Create new Project
-        shell: pwsh
-        run: ./test/Scripts.Integration.Test/integration-create-project.ps1 "${{ env.UNITY_PATH }}"
-
-      - name: Integration Test - Build Standalone Player without Sentry SDK
-        shell: pwsh
-        run: ./test/Scripts.Integration.Test/integration-build-project.ps1 "${{ env.UNITY_PATH }}"
-
-      - name: Integration Test - Add Sentry to test project
-        shell: pwsh
-        run: ./test/Scripts.Integration.Test/integration-update-sentry.ps1 "${{ env.UNITY_PATH }}"
-
-      - name: Integration Test - Build Standalone Player Sentry SDK
-        shell: pwsh
-        run: ./test/Scripts.Integration.Test/integration-build-project.ps1 "${{ env.UNITY_PATH }}"
-
-      - name: Integration Test - Run Player - Smoke Test
-        shell: pwsh
-        run: ./test/Scripts.Integration.Test/integration-run-smoke-test.ps1
-
```

Comment:
moved to a separate `windows-smoke-test` job below

Reply:
We need a Mac and Linux too for standalone. Less relevant until we have native support

Reply:
yes, seemed a bit pointless to run on those while they're not officially supported yet. Would just take away from the available github-actions runners' pool. So if you don't mind I'd also add them later when needed.